src/main/java/net/countercraft/movecraft/listener/InteractListener.java

```diff
@@ -597,6 +597,71 @@ public void run() {
 					event.getPlayer().sendMessage( String.format( I18nSupport.getInternationalisedString( "Insufficient Permissions" ) ) );
 				}
 			}
+		} else if (org.bukkit.ChatColor.stripColor(sign.getLine( 0 )).equalsIgnoreCase("contacts")) {
+			if (event.getPlayer().hasPermission( "movecraft." + craftTypeStr + ".contacts" )) {
+				if(CraftManager.getInstance().getCraftByPlayer(event.getPlayer())!=null ) {
+					Craft ccraft=CraftManager.getInstance().getCraftByPlayer(event.getPlayer());
+					boolean foundContact=false;
+					for (Craft tcraft : CraftManager.getInstance().getCraftsInWorld(ccraft.getW())) {
+						long cposx=ccraft.getMaxX()+ccraft.getMinX();
+						long cposy=ccraft.getMaxY()+ccraft.getMinY();
+						long cposz=ccraft.getMaxZ()+ccraft.getMinZ();
+						cposx=cposx>>1;
+						cposy=cposy>>1;
+						cposz=cposz>>1;
+						long tposx=tcraft.getMaxX()+tcraft.getMinX();
+						long tposy=tcraft.getMaxY()+tcraft.getMinY();
+						long tposz=tcraft.getMaxZ()+tcraft.getMinZ();
+						tposx=tposx>>1;
+						tposy=tposy>>1;
+						tposz=tposz>>1;
+						long diffx=cposx-tposx;
+						long diffy=cposy-tposy;
+						long diffz=cposz-tposz;
+						long distsquared=Math.abs(diffx)*Math.abs(diffx);
+						distsquared+=Math.abs(diffy)*Math.abs(diffy);
+						distsquared+=Math.abs(diffz)*Math.abs(diffz);
+						long detectionRange=0;
+						if(tposy>65) {
```

Comment:
tposy can't be compared with fixed 65. Sea level is set to 63 by default minecraft map generator, but it can be changed in custom map generator. F.e: we have sea level 83 on few maps for better user experiences with submarines.